src/bb-modules/Theme/Service.php

```diff
@@ -275,10 +275,8 @@ public function getThemes($client = true)
         $list = array();
         $path = $this->getThemesPath();
         if ($handle = opendir($path)) {
-            while (false !== ($file = readdir($handle))) {
-//**Deprecated: Array and string offset access syntax with curly braces is deprecated in /var/www/vhosts/webbhostingservices.com/httpdocs/boxbilling/src/bb-modules/Theme/Service.php on line 279 *//
+            while (false !== ($file = readdir($handle))) { 
                 if (is_dir($path . DIRECTORY_SEPARATOR . $file) && $file(0) != '.') {
```

Comment:
Since $file is a string, **$file(0)** should be **$file[0]**.

$file is accessed as an array hence the square bracket.

An alternative to $file[0] would be to use substr($file, 0, 1)